maestro/demos/workflows/summary.ai/workflow.yaml

```diff
@@ -10,6 +10,15 @@ spec:
       name: summary-ai
       labels:
         project: maestro-demo
+    event:
+      cron: "0 0 * * *"
+      name: daily run
+      steps:
+        - retreive papers
+        - choose paper
+        - retrieve abstract
+        - get summary
+      exit: "True"
```

Comment:
This doesn't make it repeat everyday.  It always terminates this workflow. Is it what you intent?